scripts/mbedtls_framework/c_parsing_helper.py

```diff
@@ -60,18 +62,39 @@ def __init__(self, #pylint: disable=too-many-arguments
                  qualifiers: Iterable[str],
                  return_type: str,
                  name: str,
-                 arguments: List[str]) -> None:
+                 arguments: List[str],
+                 doc: str = "") -> None:
+
         self.filename = filename
         self.line_number = line_number
         self.qualifiers = frozenset(qualifiers)
         self.return_type = return_type
         self.name = name
         self.arguments = [ArgumentInfo(arg) for arg in arguments]
+        self.doc = doc
 
     def returns_void(self) -> bool:
         """Whether the function returns void."""
         return bool(self.VOID_RE.search(self.return_type))
 
+    def __str__(self) -> str:
+        str_args = [str(a) for a in self.arguments]
+        str_text = "{} {} {}({})".format(" ".join(self.qualifiers),
+                                         self.return_type, self.name,
+                                         ", ".join(str_args)).strip()
+        str_text = FunctionInfo._c_wrap__(str_text)
+        return self.doc + "\n" + str_text
+
+    @staticmethod
+    def _c_wrap__(in_str: str, line_len: int = 80) -> str:
+        """ Will wrap function  strings over line_len length and
```

Comment:
Going by the context, this function is used to format function prototypes. Why not keep it simple and put one argument per line, like we're doing in the C sources? That avoids varying indentation based on the function name and zigzagging to read parameters in arbitrary groups.

Reply:
This is placing an argument per line as we do in the source code ?

Namely it replaces normal commas with a `" " * pad_no + ",\n" `in the `in_str = in_str.replace(",", padded_comma)` method.